tracer/test/test-applications/integrations/Samples.Amazon.Lambda.RuntimeSupport/serverless.lambda.dockerfile

```diff
@@ -1,31 +0,0 @@
-ARG lambdaBaseImage
```

Comment:
Was it deleted on purpose? It doesn't seem related to the PR

Reply:
Yeah, it's a duplicate of the _other_ `serverless.lambda.dockerfile` which moved into the `tracer/build/_build/docker` folder with all the other dockerfiles